tattoo.ts

```diff
@@ -58,8 +58,16 @@ const cli = cliArgs([
     multiple: true,
     alias: "r",
     description:
-        "Explicit repos to process. Specifying explicit repos will disable" +
+        "Explicit repos to load. Specifying explicit repos will disable" +
         "running on the implicit set of repos for the user."
+  }, {
+    name: "test-repo",
+    type: String,
+    defaultValue: [],
+    multiple: true,
+    alias: "t",
+    description:
+        "Repositories to test."
```

Comment:
should you reference --repo here? Do dependencies need to be specified with --repo or will they be bower/npm installed?